packages/dx-react-scheduler-material-ui/src/templates/views/common/time-table/layout-container.jsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import RootRef from '@material-ui/core/RootRef';
+
+export class TimeTableContainer extends React.PureComponent {
```

Comment:
Since component has `children` property (which most likely will be different on each render) it should extend `React.Component` (not `React.PureComponent`).